app.js

```diff
@@ -543,9 +544,9 @@ d.run(() => {
 			db(cb) {
 				var db = require('./db');
 				db
-					.connect(config.db, dbLogger)
+					.connect(config.db, dbLogger, logger)
```

Comment:
No need to pass two logger. Just db specific logger is enough for `db` to log any thing.

Reply:
`dbLogger` is used for logging to database log file, where as `logger` is used for logging incase of an `error` which later mighht causes application cleanup. [here](https://github.com/LiskHQ/lisk/pull/1919/files/f575473e78e9a180ff2a1363193064fc829f8020#diff-b13c80ec549dd76940c7f534e5d7f290R97)